### DISCUSSION TITLE
Assertion of sequence state in Adjacency

### DISCUSSION THREAD

**Discussion by rrfreimuth** [ga4gh/vrs#552] [Q&A]
When defining an Adjacency, is the state of the sequence extending away from the junction asserted to match the reference used to define the location(s)? The linker sequence (if present) is presumed to be known with some level of precision so I am trying to understand the extent to which that knowledge of state is conveyed to the regions flanking the Adjacency.

**Comment by larrybabb** [comment]
Like any `SequenceLocation` in VRS we do NOT make any assumptions about the whether the sequence from the underlying `Sequence` within the start/end range is precise or not. We only presume the system/individual defining the `SequenceLocation` has confidence that it/they know which reference `Sequence` the location represents. `LiteralSequenceExpression`s on the other hand are precise and typically very short spans of nucleotides.
